README.md

```diff
@@ -55,5 +55,5 @@ License
 
 
  [1]: http://square.github.io/picasso/
- [2]: https://search.maven.org/remote_content?g=com.squareup.picasso&a=picasso&v=LATEST
+ [2]: https://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.squareup.picasso%22%20AND%20a%3A%22picasso%22
```

Comment:
should this go directly to https://search.maven.org/remotecontent?filepath=com/squareup/picasso/picasso/2.71828/picasso-2.71828.aar

Reply:
No, because then it'll have to be updated every release...

Reply:
But are people smart enough to click 'aar' on the right? My guess is 65% yes, 35% no.

Reply:
yeah, my thought was it can change to https://search.maven.org/remote_content?g=com.squareup.picasso33&a=picasso&v=LATEST when there is a Picasso 3 release and leave the direct, hardcoded link for the single Picasso e release.

nbd, though haha!

Reply:
that still wouldn't work.  `remote_content` seems to assume a jar, unless there's a packaging query parameter that I can't figure out...

Reply:
Yeah I'm sure we have it somewhere on another project. I like the query though as it shows all versions

Reply:
👍